docs/core/tools/dotnet-nuget-add-source.md

```diff
@@ -25,6 +25,9 @@ dotnet nuget add source -h|--help
 
 The `dotnet nuget add source` command adds a new package source to your NuGet configuration files.
 
+> [IMPORTANT]
+> Avoid having the public `nuget.org` feed along with other feeds. For more information, see [Dependency confusion attack](https://www.msn.com/money/other/microsoft-warns-enterprises-of-new-dependency-confusion-attack-technique/ar-BB1dynJl).
```

Comment:
This link is invalid, doesn't resolve for me. I did find related articles, I'm not sure this is directly related though.

Reply:
@IEvangelist See also https://github.com/dotnet/docs/pull/22777.